### DISCUSSION TITLE
ArchiveIT Captures Not Working

### DISCUSSION THREAD

**Discussion by vkern** [ManifoldScholar/manifold#3576] [Q&A]
We have been experimenting with [Archive IT](https://archive-it.org/) captures of Manifold materials but Archive IT struggles with playback. We've checked in with Internet Archiving surround the playback issues we're having. They responded with "I think this is because clicking on the links is activating [POST requests](https://urldefense.com/v3/__https:/support.archive-it.org/hc/en-us/articles/10564834618388-Troubleshooting-POST-Requests__;!!K-Hz7m0Vt54!iTBLgpXGYsP4hvGeTuhmz7yeQgWPU2oC3jX_sapUypfAg4-gtShSOmN1X-2XdngGUQBYNIi5kMYZdL2WD-Yoe7KV$) under the hood. POST requests are an HTTP request-response method that is difficult for our crawling technology to capture and replay. Archive IT is one tool we use to take a snapshot and preserve digital scholarship we create. It would be great if it could work with Manifold.

**Comment by zdavis** [comment]
I'm not sure the explanation here is really accurate. Links in Manifold typically don't generate POST requests, as a peek at the network pain of the browser inspector will surely confirm. That said, Manifold is a SPA (single-page-app) with server-side and client-side rendering, and it's possible that Archive IT isn't sophisticated enough to capture this kind of application.

**Comment by vkern** [comment]
We ended up deleting our Archive IT test as we're always watching our space quota. It might be best to connect directly to Archive IT to have a conversation as to why this isn't working. I'm not sure we're good middle people to relay back and forth! We contacted them [here](https://www.archive-it.org/contact-us). It is a pretty commonly used tool for capturing and cataloging archival copies of sites in Libraries so I can't imagine we would be the only people interested in using Archive IT as an archival solution for Manifold projects.
